test/hack/cleanup/main.go

```diff
@@ -52,7 +54,7 @@ const (
 
 type CleanableResourceType interface {
 	Type() string
-	Get(context.Context, time.Time) ([]string, error)
+	Get(context.Context, string, time.Time) ([]string, error)
```

Comment:
The cluster name is global, right? Should this be some value that's updated in a global variable that can be accessed from everywhere? Or maybe some value that's injected into the context?